### ISSUE TITLE
Server null ref: T2.NetworkMessage_RestartGame.ProcessReceivedPacket / T2.NetworkServer.SendWriterToLiteNetPeer

### ISSUE THREAD

**Issue by ChrisNZL** [ChrisNZL/Tallowmere2#733]
From 0.2.1.

Error on server when party is trying to restart game after death.

Perhaps a peer has disconnected and a null reference is happening due to a player disconnecting (not wanting to stay in the game)?

```
9:55:55, Frame 2687537, EXCEPTION »  NullReferenceException: Object reference not set to an instance of an object
>>>>> CRITICAL ERROR >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

T2.NetworkServer.SendWriterToLiteNetPeer (LiteNetLib.NetPeer liteNetPeer, LiteNetLib.SendOptions options)
T2.NetworkMessage.SendToLiteNetPeer (LiteNetLib.NetPeer liteNetPeer, T2.INetworkMessageStruct messageStruct)
T2.NetworkMessage_RestartGame.ProcessReceivedPacket (T2.NetworkPeer recipient, LiteNetLib.NetPeer sender, LiteNetLib.Utils.NetDataReader reader, T2.GameStruct gameStruct)
T2.NetworkPeer.OnNetworkReceive (LiteNetLib.NetPeer peerThatSentMessage, LiteNetLib.Utils.NetDataReader reader)
LiteNetLib.NetManager.ProcessEvent (LiteNetLib.NetManager+NetEvent evt)
LiteNetLib.NetManager.PollEvents ()
T2.NetworkServer.Update ()
```

**Comment by ChrisNZL** [comment]
Still present in 0.2.3. Need to fix.

**Comment by ChrisNZL** [comment]
Should be fixed in 0.2.6.

Added null check against `liteNetPeer`.
